activity_main.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+
+    <ImageView
+        android:id="@+id/imageView"
+        android:layout_width="158dp"
+        android:layout_height="228dp"
+        android:src="@drawable/ic_android_black_24dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.498"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.395"
+        android:contentDescription="@string/todo" />
+
+    <TextView
+        android:id="@+id/textView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/robotalk"
+        android:textColor="#000000"

```

Comment:
add color in colors.xml

Reply:
So I can add any color of my choice or a particular color

Reply:
#00000 stands for white, you should add that in colors.xml(predefined file) inside of values folder adjacent to strings.xml

Reply:
I checked your code 
Files like activity_text_to_speech and activity_speech_to_text also contains the code 
android:textColor="#value" So do you want me to add all these values from different xml files to the color.xml file

Reply:
Absolutely, if needed,create a new issue and solve

Reply:
I think I have done everything as per your guidance.Now if you find it right then please merge my pull request.

Reply:
Last thing, please attach a screenshot for landscape mode(both old and new), so that difference is visible